README.md

```diff
@@ -1,11 +1,55 @@
-NumFOCUS Elections
-==================
+# NumFOCUS Board Elections
 
-This repository holds issues and documents related to the running of elections
-of the NumFOCUS Board.
+NumFOCUS has a Board of Directors who provide guidance, oversight, and support for various NumFOCUS-led initiatives.[^1]
+Board members have fixed term limits. As members complete their term or step-down, new members need to be elected to the board. Historically, there has not been a formal process for recruiting and admitting new board members.
```

Comment:
In my experience with Markdown, a single sentence on a single line improves maintainability.
```suggestion
Board members have fixed term limits. As members complete their term or step-down, new members need to be elected to the board.
Historically, there has not been a formal process for recruiting and admitting new board members.
```

Reply:
Agreed, updated!

Reply:
resolved in d3a83bd96c3514bd3d72ba2c44cbd26e166a0743.